stbt/android.py

```diff
@@ -0,0 +1,406 @@
+# -*- coding: utf-8 -*-
+
+"""Python module to control Android phones via `ADB`_ from Stb-tester scripts.
+
+Copyright © 2017 Stb-tester.com Ltd.
+License: LGPL v2.1 or (at your option) any later version (see
+https://github.com/stb-tester/stb-tester/blob/master/LICENSE for details).
+
+Usage::
+
+    from stbt.android import AdbDevice
+    adb = AdbDevice()
+    adb.tap(x=100, y=50)
+
+For feedback (video from the Android device-under-test) you can use:
+
+* Stb-tester's standard HDMI video-capture if the device-under-test is an
+  Android set-top box.
+* HDMI video-capture from a tablet or phone with an MHL adapter.
+* A camera pointed at the phone's screen with an `Stb-tester CAMERA`_ device.
+* Screenshots captured over USB via ADB.
+
+For more details on the benefits and trade-offs of each method, see
+<https://stb-tester.com/blog/2017/02/21/testing-video-playback-on-mobile-devices>.
+
+Note that you can instead use Stb-tester features such as image-matching
+in your existing Selenium/WebDriver/Appium tests. See
+<https://stb-tester.com/blog/2016/09/20/add-visual-verification-to-your-selenium-tests-with-stb-tester>.
+
+.. _ADB: https://developer.android.com/studio/command-line/adb.html
+.. _Stb-tester CAMERA: https://stb-tester.com/videos/the-stb-tester-camera-for-tvs-and-mobile-devices
+
+"""
+
+from __future__ import division
+
+import re
+import subprocess
+import sys
+import time
+from collections import namedtuple
+
+import cv2
```

Comment:
I'd prefer if you just import cv2 in the screenshot function.  That way this module is usable without that dependency.